integration/spark/src/test/common/java/io/openlineage/spark/agent/OpenLineageRunEventTest.java

```diff
@@ -56,7 +56,7 @@ public void testSerializeRunEvent() throws IOException, URISyntaxException {
     OpenLineage.SQLJobFacet sqlJobFacet = ol.newSQLJobFacet("SELECT * FROM test");
 
     OpenLineage.JobFacets jobFacets =
-        ol.newJobFacets(sourceCodeLocationJobFacet, sqlJobFacet, documentationJobFacet);
```

Comment:
Ordering of method params has changed. This shall be fixed, after @mobuchowski change (https://github.com/OpenLineage/OpenLineage/pull/528/files#diff-167945bcf8baceb61c80eee394a8fd270cffd87440ec56dff11d54b2288dd32eR59) which rewrite it builder methods.